docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
-anonymous proxy attached to them. In the case that the multisig account wanted to add or remove
-Alice or Bob or even add in a new account into the anonymous proxy, the anonymous proxy would take
-care of that change. If a multisig wanted to modify itself without an anonymous proxy, a whole new
-multisig would be created.
+As the term suggests, nested proxy calls are proxy calls within proxy calls. Such calls are needed if there are proxied accounts that are proxies themselves. In the example diagram below, Alice has a stash account that has a staking proxy account, Anon. Anon is an anonymous proxy, a proxied account originally spawned by Charly that is now a any proxy of Anon and signs everything on its behalf.
 
-![anonymous multisig proxy](../assets/multisig_proxy_diagram.png)
+![nested proxy calls 2x](../assets/nested_proxy_calls_2x.png)
 
-## How to set up a Proxy
+In this case, for example to bond more funds, Charly needs to submit a `prox.proxy` extrinsic to Anon, which in turn submits a `proxy.proxy` extrinsic to Alice including a `staking.bondExtra` extrinsic call, specifying the number of tokens that need to be bounded. If Charly wants to leave, a new account can take his place as any proxy (before Charly leaves!). There is no need to change the staking proxy account. Also, Alice is the only one who can remove Anon as a staking proxy, and Anon can only perform staking-related tasks. For example, Anon cannot send funds out from Alice's account.
 
-### Using the Polkadot-JS UI
+:::note
 
-- [Creating a Proxy](https://support.polkadot.network/support/solutions/articles/65000182179-how-to-create-a-proxy-account)
-- [Creating an Anonymous Proxy](https://support.polkadot.network/support/solutions/articles/65000182196)
+Proxy calls can be done using the Extrinsic Tab in the Polkadot-JS UI. Nested proxy calls can be done by calling each `proxy.proxy` extrinsic separately, or in some cases by just calling the last `proxy.proxy` extrinsic. In the diagram above, submitting Extrinsic (1.2) will automatically ask for the signature of Extrinsic (1.1). In the diagram below, submitting Extrinsic (1.3) will still ask for the signature of Extrinsic (1.2) and not (1.1). If you wish to sign with Bob's account you need to manually submit all three extrinsics.
 
-### Removing Proxies
+:::
 
-Under the [accounts tab in the Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts) there is blue button next to the account that has proxies.
-Hovering on the blue button lets you click on a link that says "proxy overview", which displays a pop-up 
-window like the one shown below. In this pop-up window, you have an option to clear individual proxy accounts
-or all of them. Under the hood, the UI is calling the extrinsics `removeProxy` for individual accounts and
- `removeProxies` for clearing all of the proxy accounts.
+![nested proxy calls 3x](../assets/nested_proxy_calls_3x.png)
 
-![Remove Proxies](../assets/remove_proxies.png)
+## Anonymous Proxies
```

Comment:
Add an info not here mentioning that they are now called Pure Proxies and the reasoning/reference links

Reply:
Done